dashboard/src/components/ui/v2/Switch/Switch.stories.tsx

```diff
@@ -11,7 +11,7 @@ export default {
 const Template: ComponentStory<typeof Switch> = function Template(
   args: SwitchProps,
 ) {
-  return <Switch {...args} />;
+  return <Switch label="Accept Rules" {...args} />;
```

Comment:
Shouldn't `label="Accept Rules"` be passed as `args` as this is an abstract component?

Reply:
This is just a default label for the `Switch` that's shared between stories. Users can override it through the storybook.